book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rental << rental
```

Comment:
- [ ] You are trying to push `rental` into `@rental` array that doesn't exist. Perusing through your code, I can see that you meant to write instead:
```ruby
def add_rental(rental)
  @rentals << rental
  rental.book = self
end
```
Please make this minor change as it has the potential to break your code 👍